src/main/java/com/suse/saltstack/netapi/client/SaltStackClient.java

```diff
@@ -314,6 +315,33 @@ public Job call() throws SaltStackException {
     }
 
     /**
+     * Get previously run jobs.
+     * @return map containing run jobs keyed by job id.
+     * @throws SaltStackException if anything goes wrong
+     */
+    public Map<String, Job> getJobs() throws SaltStackException {
+         Result<List<Map<String, Job>>> result = connectionFactory
+                .create("/jobs", JsonParser.JOBS, config)
+                .getResult();
+         return result.getResult().get(0);
```

Comment:
Alright, @moio  there seems to be always at least one element, if I find this correctly:
https://github.com/saltstack/salt-api/blob/develop/saltapi/netapi/rest_cherrypy/app.py#L909

But I still find this pattern a bit ugly and `result.getFirst()` is a little bit better, IMO. :-)